README.md

```diff
@@ -170,9 +170,9 @@ Wanna help with development, or just see how Nigthscout works? Great! See [CONTR
 # Usage
 
 The data being uploaded from the server to the client is from a
-MongoDB server such as [mongolab][mongodb].
+MongoDB server such as [mLab][mongodb].
 
-[mongodb]: https://mongolab.com
+[mongodb]: https://mlab.com/
```

Comment:
Coulda sworn I caught this one a long time ago, but I can't find it anywhere now. Anyway, good catch. Make sure the link gets changed to `[mLab]` too though.